.github/scripts/mega-app-copy-files.sh

```diff
@@ -82,6 +82,10 @@ fi
 if [[ "$FRAMEWORK" == 'vue' ]]; then
     echo "cp templates/components/vue/App.vue mega-apps/${MEGA_APP_NAME}/src/App.vue"
     cp templates/components/vue/App.vue mega-apps/${MEGA_APP_NAME}/src/App.vue
+    echo "cp $AWS_EXPORTS_FILE mega-apps/${MEGA_APP_NAME}/src/aws-exports.js"
+    cp $AWS_EXPORTS_FILE mega-apps/${MEGA_APP_NAME}/src/aws-exports.js
+    echo "node ../.github/scripts/modifyJSON.js mega-apps/${MEGA_APP_NAME}/tsconfig.json add \"compilerOptions.allowJs true\""
+    node ../.github/scripts/modifyJSON.js mega-apps/${MEGA_APP_NAME}/tsconfig.json add "compilerOptions.allowJs true"
```

Comment:
Can't we just have a different tsconfig.json per framework (i.e. `mega-apps/${MEGA_APP_NAME}/${FRAMEWORK}/tsconfig.json`)? Having a script that traverses the json seems unnecessary and am concerned with maintaining it.

Reply:
"have a different tsconfig.json per framework" is our current solution, which is problematic as mentioned in the PR description.

Also, it doesn't traverse the json file, it convert the json file to a Javascript object.

I think adding or removing a key from the JSON file is more like a user behavior.

If we could have modified the file earlier, we might avoid several failures because the `create-xxx` library changed their default config, which made our template outdated.

Reply:
Got it, super valid. Concerned with maintainability of complex-ish scripts with no testing. Let's add more input validation and console.logs. I'll add comments below.

Reply:
Have you considered using 3p packages like `json`? e.g.

```bash
# might need to strip the comment first
json -I -f package.json -e "this.compilerOptions.allowJs=true"
```

Reply:
That's a great idea. I didn't consider using a 3p package since it's a pretty simple script, but yea, `json` looks like a solid option. I'll update the PR using that.

Reply:
Done 4c3eae26d20047ca7bc24bfa626b1802aab1f0ec